TombLib/TombLib/LevelData/IO/Prj2Chunks.cs

```diff
@@ -115,8 +115,12 @@ internal static class Prj2Chunks
         /******/public static readonly ChunkId RoomSectors = ChunkId.FromString("TeSecs");
         /**********/public static readonly ChunkId Sector = ChunkId.FromString("TeS");
         /**************/public static readonly ChunkId SectorProperties = new ChunkId(new byte[] { 0 }); // These chunks occur very often, this minimizes their size impact
-        /**************/public static readonly ChunkId SectorFloor = new ChunkId(new byte[] { 1 });
-        /**************/public static readonly ChunkId SectorCeiling = new ChunkId(new byte[] { 2 });
+        /**************/public static readonly ChunkId SectorFloorClassic = new ChunkId(new byte[] { 1 }); // DEPRECATED
```

Comment:
I think it's not a good idea to rename existing chunks.

Reply:
The names don't hold any value for old prj2, the only thing that identifies it is the ID, the name is purely cosmetic for the code.

Reply:
Prior to your changes, chunks were never renamed. I recommend to stick to that convention.